flink-runtime/src/test/java/org/apache/flink/runtime/scheduler/DefaultExecutionSlotAllocatorTest.java

```diff
@@ -189,7 +192,7 @@ private DefaultExecutionSlotAllocator createExecutionSlotAllocator(InputsLocatio
 		return schedulingRequirements;
 	}
 
-	private SlotExecutionVertexAssignment findSlotAssignmentByExecutionVertexId(
+	static SlotExecutionVertexAssignment findSlotAssignmentByExecutionVertexId(
```

Comment:
nit: would be nice to have a separate static util class for the shared static methods

Reply:
Agreed. Added util classes in 216a75a34f2d090d726673dc272ea2801fa6eef8 and 277ee720dee5ab63dade29c7178d9e7af69a3f21